testsuite/tests/glbc/test_ingress_reconciliation.py

```diff
@@ -0,0 +1,42 @@
+"""Basic tests for ingress reconciliation"""
+from urllib.parse import urlparse
+
+import backoff
+import httpx
+import pytest
+
+from testsuite.openshift.objects.ingress import Ingress
+
+
+@pytest.fixture(scope="module")
+def backend_ingress(request, backend, blame):
+    """Returns created ingress for given backend"""
+    with backend.openshift.context:
+        service = backend.httpbin_objects.narrow("service").object()
+    service_name = service.name()
+    ports = service.model.spec.ports
+
+    name = blame("backend-ingress")
+    port = ports[0].get("port")
+
+    ingress = Ingress.create_service_ingress(backend.openshift, name, service_name, port)
+    request.addfinalizer(ingress.delete)
+    ingress.commit()
+
+    return ingress
+
+
+@backoff.on_exception(backoff.fibo, exception=(KeyError, AssertionError, httpx.ConnectError), max_time=60)
+def test_ingress_host_add(backend_ingress):
+    """Creates ingress for a backend and checks for host field filled by glbc, checks host points to backend"""
+    backend_ingress.refresh()  # upon backoff we want to refresh ingress model for changes
```

Comment:
no there isn't. If it would it most probably would use 'native' waiting that can wait only for status field conditions. If desired I can implement custom wait_until.

Reply:
There is until_all https://github.com/Kuadrant/testsuite/blob/c92a08536673f5c4a14e0e1e3ab3899348bdfbba/testsuite/openshift/objects/authorino.py#L58, which exactly uses native waiting and yes you should create a waiting method using that that will wait until rules are reconciled